### PR TITLE
Add support for bash shell.

### PR DESCRIPTION
**Pull request by uncleBlobby**
Add basic command parser for bash history file

Add switch in main.go that checks for the shell type set in config and runs shell-specific parser for each case.

### CODE REVIEW

File: history/parser.go [hunk 1]
```diff
@@ -21,6 +21,8 @@ func PrintTopCommands(cfg config.Config, p Parser) {
 		log.Fatalf("Error parsing history: %v", err)
 	}
 
+	log.Printf("commands in PrintTopCommands: %#v", commands)
+
```
Comment (quentinlintz) [inline history/parser.go]: Could we remove this log statement? When I create a release it looks like Printf logs end up in the binary.

File: history/bash_parser.go [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+	if err := scanner.Err(); err != nil {
+		return nil, err
+	}
+
+	var commands []models.Command
+	for name, count := range commandMap {
+		commands = append(commands, models.Command{Name: name, Count: count})
+	}
+
+	log.Printf("commands: %#v", commands)
```
Comment (quentinlintz) [inline history/bash_parser.go]: This could be removed too. Maybe a future feature could be build tags to conditionally display log statements?

**Review by quentinlintz — CHANGES_REQUESTED** [review]
This is FANTASTIC, thank you!

Could you add a new `history/testdata/bash_history` file with this content?
```
ls -la
cd /home
git status
ls -la
git commit -m "Initial commit"
```

Then, you can add to the `tests` tables in TestParseHistory and TestHistoryFileNotFound to run the parser and check the bash parser implementation. 

I checked to make sure it parsed a bash history file 👍🏻

File: history/parser.go (hunk 1 above)
Comment (uncleBlobby) [reply history/parser.go]: My bad! I thought I had removed all my debugging logs, but I guess I missed a couple.

I'll remove those and add that test case, then I'll circle back to you. 😁

**Review by quentinlintz — CHANGES_REQUESTED** [review]
Thank you! Last thing, sorry. Please check off the bash history support in the Roadmap README section.

**Review by quentinlintz — APPROVED** [review]

**Pull request merged**